addons/pos_restaurant/tests/test_frontend.py

```diff
@@ -266,6 +267,14 @@ def test_09_combo_split_bill(self):
     def test_10_save_last_preparation_changes(self):
         self.pos_config.write({'printer_ids': False})
         self.pos_config.with_user(self.pos_admin).open_ui()
-        self.start_tour("/pos/ui?config_id=%d" % self.pos_config.id, 'SaveLastPreparationChangesTour', login="pos_admin")
-        self.assertTrue(self.pos_config.current_session_id.order_ids.last_order_preparation_change, "There should be a last order preparation change")
-        self.assertTrue("Coca" in self.pos_config.current_session_id.order_ids.last_order_preparation_change, "The last order preparation change should contain 'Coca'")
+        self.start_tour("/pos/ui?config_id=%d" % self.pos_config.id, 'SaveLastPreparationChangesTour',
+                        login="pos_admin")
+        self.assertTrue(self.pos_config.current_session_id.order_ids.last_order_preparation_change,
+                        "There should be a last order preparation change")
+        self.assertTrue("Coca" in self.pos_config.current_session_id.order_ids.last_order_preparation_change,
+                        "The last order preparation change should contain 'Coca'")
+
+    def test_11_merge_table(self):
+        self.pos_config.with_user(self.pos_admin).open_ui()
+        self.start_tour("/pos/ui?config_id=%d" % self.pos_config.id, 'MergeTableTour', login="pos_admin", watch=False,
```

Comment:
ideally, this test should be run with `login="pos_user"`

Reply:
Don't get it, all the other test is with pos_admin and when I try with pos_user i have a : "odoo.exceptions.AccessDenied: Access Denied"